Domains/4-Application/ISM/Released/IntegratedStructuralModel.02.00.00.ecschema.xml

```diff
@@ -0,0 +1,2755 @@
+        <ECProperty propertyName="type" typeName="IsmGeometryType" description="Type of geometry defined in ISM API format." />
+        <ECArrayProperty propertyName="geometry" typeName="Bentley.Geometry.Common.IGeometry" displayLabel="Geometry" kindOfQuantity="AECU:LENGTH" description="An array of geometries representing more complex geometry." />
+    </ECStructClass>
+
+    <ECRelationshipClass typeName="IsmObject_Object" modifier="None" strength="referencing" strengthDirection="forward">
+        <BaseClass>bis:ElementRefersToElements</BaseClass>
+        <Source multiplicity="(0..*)" roleLabel="has IsmObject" polymorphic="true">
+            <Class class="bis:Element" />
+        </Source>
+        <Target multiplicity="(0..*)" roleLabel="is used by IsmObject" polymorphic="true">
+            <Class class="bis:Element" />
+        </Target>
+    </ECRelationshipClass>
+
+    <ECEntityClass typeName="IIsmPropertyHolder" modifier="Abstract">
+        <ECCustomAttributes>
+            <IsMixin xmlns="CoreCustomAttributes.01.00.03">
+                <AppliesToEntityClass>bis:Element</AppliesToEntityClass>
+            </IsMixin>
+        </ECCustomAttributes>
+    </ECEntityClass>
+
+    <PropertyCategory typeName="IIsmObject_category" displayLabel="Object" priority="200" />
+    <ECEntityClass typeName="IIsmObject" displayLabel="Ism Object" modifier="Abstract">
+        <BaseClass>IIsmPropertyHolder</BaseClass>
+        <ECCustomAttributes>
+            <IsMixin xmlns="CoreCustomAttributes.01.00.03">
+                <AppliesToEntityClass>bis:Element</AppliesToEntityClass>
+            </IsMixin>
+        </ECCustomAttributes>
+        <ECProperty propertyName="CustomProperties" typeName="string" displayLabel="Custom Property Values" category="IIsmObject_category" />
+        <ECProperty propertyName="AutoNavigatorTransparency" typeName="double" category="IIsmObject_category" description="A legacy property required for Local based Synchronizer to work." />
+        <ECProperty propertyName="ShortId" typeName="string" displayLabel="Short Id" category="IIsmObject_category" />
+        <ECProperty propertyName="Name" typeName="string" category="IIsmObject_category" description="User defined Element name." />
+        <ECProperty propertyName="IsmId" typeName="string" category="IIsmObject_category" description="Internal id used for mapping iTwin and Consuming Application Elements." />
+    </ECEntityClass>
+
+    <ECEntityClass typeName="IsmAnalysisDesign" displayLabel="Analysis Design" modifier="None">
+        <BaseClass>bis:InformationRecordElement</BaseClass>
+        <BaseClass>IIsmObject</BaseClass>
+    </ECEntityClass>
+
+    <ECRelationshipClass typeName="IsmAnalysisResults_AnalysisDesign" modifier="None" strength="referencing" strengthDirection="forward">
+        <BaseClass>IsmObject_Object</BaseClass>
+        <Source multiplicity="(0..1)" roleLabel="belongs to analysis results" polymorphic="false">
```

Comment:
If you are going to retain the other endpoint in the roleLabel this role label incorrectly lists itself instead of the other endpoint.